Fix delta import commit notification

zoliMolnar opened the pull request:
This pull request fixes the "no commit notification" after delta import is finished bug.

Commit:
[snomed] Added tests to test timestamp updating after delta import.

Commit:
[snomed] Removed extra line.

Commit:
[snomed] Added commit notification after import is successfully finished

Commit:
[snomed]Moved check inside try block, moved setCommitTime inside ifblock

Commit:
[snomed] Added delta import for test.

Commit:
[rf2] Removed string format.

nagyo commented on snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/japi/branches/BranchTimestampUpdateTest.java:
```diff
@@ -0,0 +1,83 @@
+	}
+	
+	@Test
+	public void testHeadTimestampUpdateAfterImporting() {
```
Please move this to com.b2international.snowowl.snomed.api.rest.io.SnomedImportApiTest. There is no need to have this separately from all the other import tests.

nagyo replied:
You'll have everything you need available in that package, both the import archive and all the supporting methods. In the end you can remove this class and the duplicate archive as well.

nagyo commented on snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/util/ImportUtil.java:
```diff
@@ -437,31 +441,40 @@ public void run() {
+			long commitTime = CDOServerUtils.getLastCommitTime(branch);
+			importContext.setCommitTime(commitTime);
+			
+			final SnomedCompositeImporter importer = new SnomedCompositeImporter(IMPORT_LOGGER, repositoryState, importContext, importers, ComponentImportUnit.ORDERING);
```
Why have you changed these three lines?

Commit:
[rf2] Removed BranchTimeStampUpdateTest and moved its contents to...

SnomedImportApiTest. Added new overload for importArchive, added new RF2
archive for testing and removed duplicate RF2 archive.

Commit:
[rf2] Added final modifier for commitTime local variable.

cmark commented on snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/io/SnomedImportApiTest.java:
```diff
@@ -203,5 +226,41 @@ public void import11ExtensionConceptWithVersion() {
+		getComponent(path, SnomedComponentType.CONCEPT, "301795004").statusCode(200);
+		
+		
+		assertThat(branchBeforeImport.baseTimestamp(), not(equalTo(branchAfterImport.headTimestamp())));
```
Use a shorter version of assertion to increase readability: `assertNotEquals(branchAfterImport.baseTimestamp(), branchAfterImport.headTimestamp())`.

Commit:
SO-2903: Changed assertion to improve readability

nagyo committed:
[snomed.import] Revert formatting changes

nagyo committed:
Merge remote-tracking branch 'origin/6.x' into issue/fix_delta_import_commit_notification
